Rakefile

```diff
@@ -1,6 +1,9 @@
-require "bundler/gem_tasks"
+require 'bundler/gem_tasks'
 require 'rspec/core/rake_task'
+require 'rubocop/rake_task'
+
+RuboCop::RakeTask.new
 
 RSpec::Core::RakeTask.new(:spec)
 
-task :default => :spec
+task default: %w[spec rubocop]
```

Comment:
@titusfortner this change to the default rake task runs rubocop on each travis run